ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -978,5 +978,320 @@
     "CaseEventID": "addCaseNote",
     "UserRole": "caseworker-publiclaw-judiciary",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "uploadStandardDirectionsAfterGatekeeping",
+    "UserRole": "caseworker-publiclaw-solicitor",
```

Comment:
thb, not sure if this is about this user role, It will cause that e.g barrister and la barrister will see these events as well

Reply:
It's probably ok, but waiting for confirmation from business.

Reply:
It was confirmed by business that it's about this user role - the note got added to the ticket, so resolving.